modules/openxr/scene/openxr_composition_layer.h

```diff
@@ -46,13 +46,17 @@ class OpenXRCompositionLayer : public Node3D {
 	GDCLASS(OpenXRCompositionLayer, Node3D);
 
 	SubViewport *layer_viewport = nullptr;
+	bool enable_hole_punch = false;
 	MeshInstance3D *fallback = nullptr;
 	bool should_update_fallback_mesh = false;
+	bool running = false;
```

Comment:
Bit of a nit but personally I'd like a more descriptive property name, maybe `openxr_session_running`?

Reply:
Makes sense! Updated in my latest push